deps/wazuh_testing/wazuh_testing/modules/aws/event_monitor.py

```diff
@@ -76,8 +82,11 @@ def check_processed_logs_from_output(command_output: str, expected_results: int
     )
 
 
-def check_non_processed_logs_from_output(command_output: str, expected_results: int = 1):
-    pattern = r'.*DEBUG: \+\+\+ No logs to process in bucket: '
+def check_non_processed_logs_from_output(command_output: str, bucket_type: str, expected_results: int = 1):
```

Comment:
Docstring is missing

Reply:
added in 11868d3ed5858c2f1c0f4f63bd254d45766c0565